contentcuration/contentcuration/frontend/channelEdit/components/AssessmentTab/AssessmentTab.vue

```diff
@@ -111,15 +108,7 @@
         return this.$tr('incompleteItemsCountMessage', { invalidItemsCount });
       },
     },
-    watch: {
-      assessmentItems(newAssessmentItems) {
```

Comment:
Although I am not 100% sure, I assume that this watcher might be the cause of some inconsistencies we've been experiencing lately because (1) it updates a content node in another tick, and (2) the update is run in another transaction than updates that triggered `complete` re-evaluation.

I haven't closed any related issues I am assigned to though. I'd like to test everything at once after @micahscopes is done with his updates addressing some other, but possibly related, problems.